internal/app/synchronizer/evaluator_client.go

```diff
@@ -75,8 +76,12 @@ func (ec *evaluatorClient) initialize() error {
 		return nil
 	}
 
+	evaluatorHostName := ec.cfg.GetString(internal.EvaluatorPrefix + internal.HostNameSuffix)
```

Comment:
Similar comment on first class constants over string append. Elsewhere in the file too.